root/static/scripts/common/components/EntityLink.js

```diff
@@ -72,16 +78,17 @@ const AreaDisambiguation = ({area}: {|+area: AreaT|}) => {
   }
 
   return <Comment className="historical" comment={comment} />;
-}
+};
 
 const NoInfoURL = ({url, allowNew}) => (
   <Frag>
     <a href={url}>{url}</a>
     {' '}
-    <DeletedLink name={'[' + l('info') + ']'} allowNew={allowNew} />
+    <DeletedLink allowNew={allowNew} name={'[' + l('info') + ']'} />
```

Comment:
Maybe we can modify bracketed to do all 3 kinds of brackets depending on flags passed and default to round?